Campaigns/apt ta17 293a ps.txt

```diff
@@ -0,0 +1,6 @@
+// Original Sigma Rule: https://github.com/Neo23x0/sigma/blob/master/rules/apt/apt_ta17_293a_ps.yml
+// Questions via Twitter: @janvonkirchheim
+ProcessCreationEvents
+| where EventTime > ago(7d)
+| where ProcessCommandLine == "ps.exe -accepteula"
```

Comment:
`==` is case sensitive.
I suggest to use `=~` instead, to catch `PS.exe`, `pS.ExE -AcceptEula` etc.